demos/internal-developers/README.md

```diff
@@ -0,0 +1,13 @@
+# Development-targeted notebooks
```

Comment:
This could be renamed to make it clearer that this is for internal stellargraph development, eg 
```suggestion
# StellarGraph Internal Development
```